{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[tool.poetry]
+name = "{{ cookiecutter.project_slug }}"
+version = "0.1.0"
+description = "{{ cookiecutter.project_short_description }}"
+authors = ["{{ cookiecutter.full_name }} <{{ cookiecutter.email }}>"]
+license = "Proprietary"
+packages = [
+    { include = "{{ cookiecutter.module_name }}", from = "src" },
+]
+include = ["src/{{ cookiecutter.module_name }}/res/*"]
+{% if cookiecutter.create_cli == "yes" %}
+[tool.poetry.scripts]
+{{ cookiecutter.project_slug }} = "{{ cookiecutter.module_name }}.main:app"
+{% endif %}
+[tool.poetry.dependencies]
+python = "^3.7"
+{% if cookiecutter.config_file == 'hocon' %}pyhocon = "^0.3.54"{% elif cookiecutter.config_file == 'yaml' %}PyYAML = "^5.3.1"{% endif %}
+{% if cookiecutter.create_cli == 'yes' %}typer = "^0.2.1"{% endif %}
+importlib-metadata = {version = "^1.0", python = "<3.8"}
+
+[tool.poetry.dev-dependencies]{% if cookiecutter.code_formatter == 'black' %}
+black = "^19.10b0"{% endif %}{% if cookiecutter.use_notebooks == 'true' %}
+jupyter = "^1.0.0"{% endif %}
```

Comment:
please don't use the [jupyter](https://pypi.org/project/jupyter/) meta package, it hasn't been updated in 5 years. I would leave it to the user to set up their jupyter environment

Reply:
Gotcha, my mistake